tests/forms_tests/field_tests/test_multivaluefield.py

```diff
@@ -153,3 +153,9 @@ def test_form_cleaned_data(self):
         })
         form.is_valid()
         self.assertEqual(form.cleaned_data['field1'], 'some text,JP,2007-04-25 06:24:00')
+
+    def test_disabled_has_changed(self):
+        f = MultiValueField(fields=(CharField(),
```

Comment:
Use a single line. I think two CharField()s should work just as well and make the line shorter.